changelog.d/12705.misc

```diff
@@ -0,0 +1 @@
+Complain if a federation endpoint has the `@cancellable` flag, since some of the wrapper code may not handle cancellation correctly yet.
```

Comment:
Is the point just that we haven't audited this to check that it's cancellation-safe?